crawlera_fetch/middleware.py

```diff
@@ -14,6 +14,8 @@
 from scrapy.spiders import Spider
 from scrapy.utils.reqser import request_from_dict, request_to_dict
 from w3lib.http import basic_auth_header
+import base64
+import binascii
```

Comment:
Could you move these stdlib imports up, as per [PEP8?](https://www.python.org/dev/peps/pep-0008/#imports)